app/views/changesets/show.html.erb

```diff
@@ -40,9 +40,10 @@
         <% next unless comment.visible || current_user&.moderator? %>
         <li id="c<%= comment.id %>">
           <small class='text-body-secondary'>
-            <%= t comment.visible ? ".comment_by_html" : ".hidden_comment_by_html",
+            <% comment_info = comment.visible ? "comment_by_html" : "hidden_comment_by_html" %>
+            <%= t(".#{comment_info}",
```

Comment:
Looks like once you do `t ".#{comment_info}"` i18n-tasks stops reporting `changesets.show.`-anything as missing.

You could have inserted this line with the same effect:

    <% t ".#{'shut up'}" %>